x-pack/test/security_solution_cypress/cypress/tasks/navigation.ts

```diff
@@ -91,6 +92,12 @@ export const visitUserDetailsPage = (userName = 'test') => {
   visitWithTimeRange(userDetailsUrl(userName));
 };
 
+export function goBackToRulesTableViaBreadcrumbs(): void {
+  cy.log('Navigate back to rules table via breadcrumbs');
+  cy.get(`${RULES_TABLE_BREADCRUMB}:not(${LAST_BREADCRUMB})`).click();
+  cy.get(`${RULES_TABLE_BREADCRUMB}${LAST_BREADCRUMB}`).should('exist');
+}
```

Comment:
Shouldn't this be in `tasks/breadcrumbs.ts` for consistency with `screens`?

Alternatively (and I think this would be better from the code ownership perspective), we'd have a `rule_management_page.ts` page object that would contain this function and the corresponding CSS selector. Our team would own this file. That's a separate story though, I'm not suggesting to work on this refactoring now.

Reply:
I moved it to `rules_management.ts` file, it can be renamed to `rule_management_page.ts` if necessary.